web-ui/src/main/resources/catalog/js/edit/EditorController.js

```diff
@@ -279,7 +279,7 @@
         if (tabIdentifier !== $('#currTab')[0].value) {
           gnUtilityService.scrollTo();
         }
-
+        $scope.gnCurrentEdit.tab = tabIdentifier;
```

Comment:
Is this needed, onFormLoad take care of that https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/metadatamanager/EditorService.js#L279 ?